functions/src/routes/figure.ts

```diff
@@ -1,87 +1,46 @@
 import {Request, Response} from "express";
 import {Figure} from "../models/figure";
+import {getDatabase, ref, get, child, update} from "firebase/database";
+import firebase from "../../firebase";
 
-// const figureRouter = Router();
-
-const TEMP_DB: { [key:string]: Figure } = {
-  "figure1": new Figure({
-    id: "test", name: "Melina Figure (Elden Ring)",
-    image: "image URL", price: 69.99, stock: 100, present: true,
-  }),
-  "figure2": new Figure({
-    name: "Melina Pop Figure (Elden Ring)",
-    image: "image URL", price: 39.99, stock: 50, present: true,
-  }),
-};
-
-export async function getFigures(req: Request, res: Response){
+// e.g.
+// http://localhost:5001/claymore-d6749/us-central1/default/figure?figureId=figure1
+export async function getFigures(req: Request, res: Response) {
   const figureId = req.query.figureId?.toString();
   try {
+    const db = getDatabase(firebase);
```

Comment:
Can this be called once or should it be left on each new request?

Reply:
It can be called just once, I don't see the advantage of maintaining an additional global though. Couldn't find anything about speed looking it up just now